docs/products/networking/nodebalancers/guides/manage/index.md

```diff
@@ -0,0 +1,58 @@
+---
+author:
+  name: Linode
+  email: docs@linode.com
+title: "Manage NodeBalancers"
+description: "Learn how to view, edit, and delete NodeBalancers on the Linode platform."
+published: 2022-10-06
+---
+
+## View NodeBalancers
+
+Log in to the [Cloud Manager](https://cloud.linode.com) and select NodeBalancers from the left menu. If any NodeBalancers exist on your account, they are listed on this page.
+
+![Screenshot of the NodeBalancer listing page in Cloud Manager](nodebalancer-view-list.png)
+
+Each NodeBalancer in the matrix is displayed alongside the following details:
+
+- **Backend Status:** The number of backend machines that are available and accepting connections (*up*) or have been removed from the rotation and are not accepting connections (*down*).
+- **Transferred:** The amount of network transfer consumed by the inbound traffic to the NodeBalancer.
+- **Ports:** A list of the ports that have been configured on the NodeBalancer.
+- **IP Address:** The NodeBalancer's IPv4 address.
+- **Region:** The data center where the NodeBalancer is located.
+
+## Create a NodeBalancer
```

Comment:
This section is repeated from the updated Create Nodebalancer guide. Maybe link and then remove?